content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md

```diff
@@ -0,0 +1,130 @@
+
+Suppose that your Pixie custom domain name is `pixie.example.com` and the IP address obtained above is `a.b.c.d`. Two `A` records need to be created as follows.
+```
+pixie.example.com           a.b.c.d
+work.pixie.example.com      a.b.c.d
+```
+
+## 3. Install Pixie Cloud
+
+If you are using the [Self-Hosted installation](/installing-pixie/install-guides/self-hosted-pixie/),
+1. Follow steps 1 - 5 and 8 in [Deploy Pixie cloud](/installing-pixie/install-guides/self-hosted-pixie/#1.-deploy-pixie-cloud).
+2. Comment lines 94 to 106 in `./scripts/create_cloud_secrets.sh`
+3. Execute the script as explained in step 9 of [Deploy Pixie cloud](/installing-pixie/install-guides/self-hosted-pixie/#1.-deploy-pixie-cloud)
+
+If you are using the [Air Gapped installation](/installing-pixie/install-guides/airgap-pixie/),
+1. Follow steps 1 and 4 in [Deploy Pixie Cloud](/installing-pixie/install-guides/airgap-pixie/#deploy-pixie-cloud)
+2. Comment lines 94 to 106 in `./scripts/create_cloud_secrets.sh`
+3. Execute the script as explained in step 6 of [Deploy Pixie Cloud](/installing-pixie/install-guides/airgap-pixie/#deploy-pixie-cloud)
+
+
+## 4. Create TLS certificate
+
+A TLS certificate is required for the custom domains that you wish to use with Pixie.
+
+Suppose that your Pixie custom domain name is `pixie.example.com`. You need to obtain a single certificate that is valid for both `pixie.example.com` and `work.pixie.example.com`. Finally, create a Kubernetes secret (of type `kubernetes.io/tls`) named `cloud-proxy-tls-certs` in the `plc` namespace using the certificate.
+
+One way to obtain it is by creating a Let's Encrypt certificate using [cert-manager](https://cert-manager.io/). [Securing NGINX-ingress tutorial](https://cert-manager.io/docs/tutorials/acme/nginx-ingress/) tutorial from cert-manager has detailed information about the process.
+
+Following are sample resources that you can use with cert-manager.
+
+Sample `ClusterIssuer` resource
+```yaml
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-cluster-issuer
+spec:
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory
+    email: admin@example.com
+    privateKeySecretRef:
+      name: letsencrypt-cluster-issuer-key
+    solvers:
+    - http01:
+       ingress:
+          class: nginx
+``` 
+
+Sample `Certificate` resource
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: cloud-proxy-tls-certs
+  namespace: plc
+spec:
+  dnsNames:
+  - pixie.example.com
+  - work.pixie.example.com
+  secretName: cloud-proxy-tls-certs
+  issuerRef:
+    name: letsencrypt-cluster-issuer
+    kind: ClusterIssuer
+```
+
+## 5. Create Ingress resources
+
+Two Kubernetes Ingresses are required for Pixie Cloud; One for HTTPs and the other for gRPCs communication.
+
+Create two ingresses using the following configurations
+
+1. [HTTPs]
+2. [gRPCs]
+
+```bash
+kubectl apply -f <filename>
+```
```

Comment:
We can just make this:
```bash
kubectl apply -f k8s/cloud/overlays/exposed_services_nginx/cloud_ingress_grpcs.yaml
kubectl apply -f k8s/cloud/overlays/exposed_services_nginx/cloud_ingress_https.yaml
```

Reply:
Fixed through https://github.com/pixie-io/docs.px.dev/pull/245/commits/eb0693c49c18d9f219c0275b718d4f3d82dbbf5a